ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-interfaces.j2

```diff
@@ -1,3 +1,4 @@
+{# j2lint: disable=jinja-statements-delimeter #}
```

Comment:
This template should be refactored to set temporary variables above the printout. This will also solve the issue with multiple jinja statements on the same line.